### PR TITLE
Add Polygon and Mumbai support

### PR DESCRIPTION
**Pull request by lorenzoPrimi**
Inspired from https://github.com/EthWorks/useDApp/commit/21f59f6f80bc9307abaa1dc3cb6e17230b77899f

### CODE REVIEW

File: packages/core/src/constants/chainId.ts [hunk 1]
```diff
@@ -30,13 +34,16 @@ export const MULTICALL_ADDRESSES = {
   [ChainId.Kovan]: '0x2cc8688c5f75e365aaeeb4ea8d6a480405a48d2a',
   [ChainId.BSC]: '0x41263cba59eb80dc200f3e2544eda4ed6a90e76c',
   [ChainId.xDai]: '0xb5b692a88bdfc81ca69dcb1d924f59f0413a602a',
+  [ChainId.Polygon]: '0xb5b692a88bdfc81ca69dcb1d924f59f0413a602a',
+  [ChainId.Mumbai]: '0xb5b692a88bdfc81ca69dcb1d924f59f0413a602a',
```
Comment (marekkirejczyk) [inline packages/core/src/constants/chainId.ts:38]: I don't think those are proper addresses, but I can see proper once are listed here:
https://github.com/makerdao/multicall#multicall-contract-addresses

**Comment by marekkirejczyk** [comment]
@lorenzoPrimi Good job so far! ❤️ 

I added a comment (need to fix multicall addresses) and the linter is failing.
Once both are fixed, we are ready to merge :)

File: packages/core/src/constants/chainId.ts (hunk 1 above)
Comment (lorenzoPrimi) [reply packages/core/src/constants/chainId.ts]: I was sure I entered these, need to improve my copypasta skills!
Done now

**Review by marekkirejczyk — APPROVED** [review]

**Comment by marekkirejczyk** [comment]
Looks good, still linter needs to be fixed (run `yarn lint` before commit)

**Pull request merged**